flow-nodes/api-builder-plugin-petstore/openapi/petstore.json

```diff
@@ -0,0 +1,1054 @@
+{
+  "swagger": "2.0",
```

Comment:
Don't think it matters if we want either. The version of OAS definitely doesn't need to be reflected in the plugin name since a consumer of the plugin shouldn't need to care.

Reply:
We can leave it as is.